projects/step-frontend/src/lib/modules/_common/components/grid-layout/container/gridster-config.directive.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  AfterViewInit,
+  DestroyRef,
+  Directive,
+  ElementRef,
+  Host,
+  inject,
+  Input,
+  Optional,
+  Renderer2,
+} from '@angular/core';
+import { ElementResizeDirective } from '@exense/step-core';
+import { GridsterConfig, GridsterComponent } from 'angular-gridster2';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+
+@Directive({
+  selector: '[stepGridsterConfig]', // Apply this directive on <gridster>
+  standalone: true,
+})
+export class StepGridsterConfigDirective implements AfterViewInit {
+  private _destroyRef = inject(DestroyRef);
+  @Input() options: GridsterConfig = {}; // User-provided options (optional)
```

Comment:
I don't see usages of it's input.
If it is not required, pleases remove it.
Otherwise apply it + convert it to signal